src/System.Runtime/tests/System/ActivatorTests.netcoreapp.cs

```diff
@@ -18,5 +23,201 @@ public static void CreateInstance_Invalid()
+            }
+            Assert.True(File.Exists(physicalFileName));
+        }
+
+        public static TheoryData<string, string, string, string, Type> TestingCreateInstanceFromObjectHandleData => new TheoryData<string, string, string, string, Type>()
+        {
+            //string physicalFileName, string assemblyFile, string typeName, returnedFullNameType, expectedException
+            { "TestLoadAssembly.dll", "TestLoadAssembly.dll", "PublicClassSample", "PublicClassSample", null },
+            { "TestLoadAssembly.dll", "testloadassembly.dll", "publicclasssample", "PublicClassSample", typeof(TypeLoadException) },
+
+            { "TestLoadAssembly.dll", "TestLoadAssembly.dll", "PrivateClassSample", "PrivateClassSample", null },
+            { "TestLoadAssembly.dll", "testloadassembly.dll", "privateclasssample", "PrivateClassSample", typeof(TypeLoadException) },
+
+            { "TestLoadAssembly.dll", "TestLoadAssembly.dll", "PublicClassNoDefaultConstructorSample", "PublicClassNoDefaultConstructorSample", typeof(MissingMethodException) },
+            { "TestLoadAssembly.dll", "testloadassembly.dll", "publicclassnodefaultconstructorsample", "PublicClassNoDefaultConstructorSample", typeof(TypeLoadException) }
+        };
+
+        [Theory]
+        [MemberData(nameof(TestingCreateInstanceObjectHandleData))]
+        public static void TestingCreateInstanceObjectHandle(string assemblyName, string type, string returnedFullNameType, Type exceptionType, bool returnNull)
+        {
+            ObjectHandle oh = null;
+
+            if (exceptionType != null)
+            {
+                Assert.Throws(exceptionType, () => Activator.CreateInstance(assemblyName: assemblyName, typeName: type));
+            }
+            else
+            {
+                oh = Activator.CreateInstance(assemblyName: assemblyName, typeName: type);
+                if (returnNull)
+                {
+                    Assert.Null(oh);
+                }
+                else
+                {
+                    CheckValidity(oh, returnedFullNameType);
+                }
+            }
+
+            if (exceptionType != null)
+            {
+                Assert.Throws(exceptionType, () => Activator.CreateInstance(assemblyName: assemblyName, typeName: type, null));
+            }
+            else
+            {
+                oh = Activator.CreateInstance(assemblyName: assemblyName, typeName: type, null);
+                if (returnNull)
+                {
+                    Assert.Null(oh);
+                }
+                else
+                {
+                    CheckValidity(oh, returnedFullNameType);
+                }
+            }
+        }
+
+        public static TheoryData<string, string, string, Type, bool> TestingCreateInstanceObjectHandleData => new TheoryData<string, string, string, Type, bool>()
+        {
+            //string assemblyName, string typeName, returnedFullNameType, expectedException
```

Comment:
space after ```//```

Reply:
add space after all comments

Reply:
done

Reply:
sorry for all this commit  I'm a little bit fried today :dizzy_face:

Reply:
np :)